README.md

```diff
@@ -1 +1,20 @@
-# react-tradingview-widget
\ No newline at end of file
+# react-tradingview-widget-components
+
+## Installation
+`npm i react-tradingview-widget-components`
+
+## Usage
+```typescript
+import { SymbolOverview } from "react-tradingview-widget-components";
+
+export const App () => {
+    const [ isDarkMode, setIsDarkMode ] = useState<boolean>(true);
+
+    return (
+        <SymbolOverview colorTheme={isDarkMode ? "dark" : "light"}/>
+    )
+};
+```
+
```

Comment:
definitely a good start, it would probably be good to help document (in perhaps a table) the other props that are available to the widget, or in the least provide some kind of link to the documentation to review the additional props.

some additional "fun" things you can add are stuff like https://shields.io/